### DISCUSSION TITLE
Invalid `name` parameter. (400)

### DISCUSSION THREAD

**Discussion by abduljeleelng** [vercel/vercel#4587] [Help]
I am trying to add DNS to my domain following the instruction on documentation 
`vercel dns add example.com '@' MX mx1.improvmx.com 10`
`vercel dns add example.com '@' MX mx2.improvmx.com 20`
After I replace example.com with my domain 
getting the error Invalid `name parameter. (400)`
I will appreciate any help 
Thank

**Comment by markozxuu** [comment]
Hi @abduljeleelng 👋 What hosting provider are you using?

**Reply by abduljeleelng** [reply]
Thank @markozxuu for your response, I'm using vercel and have added custom domain `wibrand.com.ng`, I was assigned nameserver and I have configure that, so I want to had DNS for email services, that's when I got the errors. I will really appreciate any support. 

thanks

**Comment by wiiiimm** [comment]
Hi @abduljeleelng are you still having this issue? What is your vercel-cli version?

I just ran the following without issue.

<img width="772" alt="image" src="https://user-images.githubusercontent.com/179761/84467015-6b6f5700-acad-11ea-8464-9a8c041b8f7c.png">



PS: Vercel has released a web based DNS editor. Can you give that a try? You can click on the "Domain" tab and select the domain from the list to get to the DNS editor UI.
